UndertaleModLib/Models/UndertaleTexturePageItem.cs

```diff
@@ -27,6 +30,13 @@ public class UndertaleTexturePageItem : UndertaleResource, INotifyPropertyChange
         private ushort _BoundingHeight;
         private UndertaleResourceById<UndertaleEmbeddedTexture, UndertaleChunkTXTR> _TexturePage = new UndertaleResourceById<UndertaleEmbeddedTexture, UndertaleChunkTXTR>();
 
+        private static int maxSpriteWidth = 0;
+        private static int maxSpriteHeight = 0;
+        private static UInt32[] pixelBuffer = null; // a memory buffer for the largest possible sprite.
```

Comment:
Keeping a large buffer forever sounds like a bad idea, and I don't really like these being cached as static in the model class anyway. It would probably be better to have a separate TextureExporter class for this that can be properly garbage collected when you are done exporting the whole batch of textures (since I assume this was for performance reasons).

And while you're at it, feel free to replace the UndertaleModTool exporting implementation with this since that one reuses WPF display code instead of doing it properly and I feel like it's still sometimes buggy. And add importing for sprites where no scaling or cropping is required since it looks way easier with this approach than the old one.

Reply:
Good point, it can't be thread-safe either if it's some static buffer like that. I'll fix that.

I won't add importing right now, but I'll add it to my TODO list.

Reply:
Importing has been added.